src/models/room.ts

```diff
@@ -2647,15 +2647,15 @@ function memberNamesToRoomName(names: string[], count = (names.length + 1)) {
     if (!names.length) {
```

Comment:
There is a `// TODO i18n` comment above we can remove now that this PR is translated it